types/cb_board_support.yaml

```
@@ -19,3 +19,25 @@ types:
       - E
       - F
       - PilotFault
+  ContactorState:
+    description: Models the state of a contactor
+    type: string
+    enum:
+      - Unknown
+      - Open
+      - Closed
+  ContactorFeedbackType:
+    description: >-
+       Models the feedback type of the auxiliary contact of the contactor:
+       none - no feedback signal is used
+       nc- the feedback signal is wired to a normally-closed contact
```

Style: missing space
```suggestion
       nc - the feedback signal is wired to a normally-closed contact
```